doc/airline.txt

```diff
@@ -1140,6 +1142,8 @@ with the middle mouse button to delete that buffer.
 * enable/disable displaying tabs, regardless of number. (c) >
   let g:airline#extensions#tabline#show_tabs = 1
 
+  Note: To display tabs instead of buffers, set `show_buffers=0` instead.
```

Comment:
I think this should be:
> Note: To display tabs instead of buffers, set `show_buffers=0` instead.

To display only tabs and no buffers, set `show_buffers=0` in addition.

Reply:
You're right, I found `show_buffers && curtabcnt == 1 || !show_tabs` now. Fixed.